### ISSUE TITLE
Layout/EmptyComment auto-correct incorrectly removes entire line

### ISSUE THREAD

**Issue by DMA57361** [bbatsov/rubocop#5674]
--------

## Expected behavior

Given the contrived example file `example.rb`:

``` ruby
def a_method #
  true
end

Rspec.describe Class do
  it 'will to a test' do #
    expect(true).to eq true
  end
end
```

When running `rubocop example.rb --only Layout/EmptyComment --auto-correct`

I expect two warnings for the two empty comments, and for them to be removed from the end of the two lines.

## Actual behavior

The lines with the empty comments are completely removed from the source:

``` ruby
  true
end

Rspec.describe Class do
    expect(true).to eq true
  end
end
```

And the output of the Rubocop execution:

```
$ rubocop example.rb --only Layout/EmptyComment --auto-correct
Inspecting 1 file
E

Offenses:

example.rb:1:14: C: [Corrected] Layout/EmptyComment: Source code comment is empty.
def a_method #
             ^
example.rb:2:1: E: Lint/Syntax: unexpected token kEND
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
end
^^^
example.rb:6:26: C: [Corrected] Layout/EmptyComment: Source code comment is empty.
  it 'will to a test' do #
                         ^
example.rb:7:1: E: Lint/Syntax: unexpected token kEND
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
end
^^^

1 file inspected, 4 offenses detected, 2 offenses corrected
```

## RuboCop version

```
$ rubocop -V
0.53.0 (using Parser 2.5.0.3, running on ruby 2.3.5 x86_64-darwin15)
```

**Comment by rrosenblum** [comment]
It looks like this cop isn't accounting for empty comments next to code at all. The auto-correction is assuming that the line is going to be blank which is what is causing the issue.
